lib/shopify-cli/js_deps.rb

```diff
@@ -59,7 +59,7 @@ def yarn(verbose = false)
     end
 
     def npm(verbose = false)
-      cmd = %w(npm install --no-audit --no-optional)
```

Comment:
@tanema looks like you added this in https://github.com/Shopify/shopify-app-cli/pull/517 – just checking if you can see any problem with removing it.